packages/webapp/pages/onboarding.tsx

```diff
@@ -228,6 +232,14 @@ export function OnboardPage(): ReactElement {
     [activeScreen],
   );
 
+  useEffect(() => {
+    // Ensure dark mode for authentication screen, since we only have a darkmode background image
+    if (!user) {
+      setTheme(ThemeMode.Dark);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
setTheme set's it globally in settings, you want to hotfix it temporary for this screen right?
I think you want to use applyTheme then like we do for the plus page alone.

https://github.com/dailydotdev/apps/blob/36d157e910b1f3cd7bbb0c96ab9e20fb980d59f0/packages/webapp/components/layouts/PlusLayout/PlusLayout.tsx#L36-L41